sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -168,6 +173,21 @@ public AzureMonitorExporterBuilder connectionString(String connectionString) {
         if (endpoint != null) {
             this.endpoint(endpoint);
         }
+        String authorizationType = keyValues.get("Authorization");
+        if (authorizationType != null && authorizationType.equals("aad") && this.credential == null) {
+            // Use user assigned managed Identity's client Id if provided
+            String clientId = keyValues.get("AppId");
+            if (clientId != null) {
```

Comment:
Any other validation needed?

Reply:
Not going through the connection string route